source/index.md

```diff
@@ -68,6 +68,28 @@ The following illustrate of what can be achieved with the Nix ecosystem:
 - Atomic upgrades and rollbacks.
 
 
+## Who is Nix for?
+
+Nix is a tool for software developers.
```

Comment:
```suggestion
Nix is primarily a tool for software developers.
```

I think adding "primarily" helps keep the door open to different audiences who may be willing enough to learn and use Nix, while not distracting from the focus on software developers.